src/outfit.ts

```diff
@@ -309,6 +327,54 @@ export class Outfit {
     return this.equipSpec(thing);
   }
 
+  bjornify(target: Familiar | Familiar[]): boolean {
+    if (this.equips.get($slot`back`) !== $item`Buddy Bjorn`) return false;
```

Comment:
Two consistent things I see here with other methods:

- The modes function allows you to set a mode even before the relevant item is equipped. So maybe it would be better to drop the check that the Bjorn is already equipped.
- If the Bjorn already has a familiar in it, this function behavior should match the `equip` behavior. So return true if the given familiar is the same as the equipped familiar, and false if they don't match (and don't change the outfit).